README.md

```diff
@@ -75,6 +75,9 @@ There are two specification extentions you can use to shape how the bindings are
 
 ```x-scs-group``` : This will add the group value on a binding which configures your microservice to use [Consumer Groups](https://cloud.spring.io/spring-cloud-static/spring-cloud-stream/current/reference/html/spring-cloud-stream.html#consumer-groups)
 
+### Limitations
+Currently any schemas that are used must be in the components/schemas part of the document. We do not support anonymous object-type schemas in the message/payload sections.
```

Comment:
what about schemas in message.payload that have `$id` provided? https://github.com/asyncapi/parser-js/blob/master/lib/models/schema.js#L23

would be good to support such approach and explain to users. Have a look how Jonas uses `$id` prop https://www.asyncapi.com/blog/designing_your_apis_with_asyncapi_part_1#introducing-reusability

Reply:
Thanks, I'll look at these possibilities.